react/features/base/react/components/native/JitsiSectionList.js

```diff
@@ -0,0 +1,97 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import {
+    SafeAreaView,
+    SectionList
+} from 'react-native';
+
+
+/**
+ * Implements a React/Web {@link Component} for displaying text similar to React
+ * Native's {@code Text} in order to faciliate cross-platform source code.
+ *
+ * @extends Component
+ */
+export default class JitsiSectionList extends Component {
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+
+    static propTypes = {
```

Comment:
Could you please use flow?

Reply:
- [] change `propTypes` to `Props` implementations and add flow